recipes/pyrfr/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "pyrfr" %}
+{% set version = "0.8.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 003d38810b3aeee4f782619afd9e039fe29a44f934788a0503749f753c6305a4
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - boost
```

Comment:
This should probably be in `host` (I assume it's using Boost-Python?)

Reply:
This comes from here: _Technically, you need Boost if you want to compile the unit tests._ at: https://github.com/automl/random_forest_run#requirements

To be honest, not sure, because it's just for compiling the tests which aren't even included in the PyPI package so it might be that it's not needed at all.

@mfeurer Maybe you can comment on how you're actually using boost? Otherwise, I could just try it in different sections and see where it fails 😄.

Reply:
@chrisburr From what I can see in the upstream repo boost is mainly used in the tests and by Cereal and not through boost-python: https://github.com/automl/random_forest_run/search?p=1&q=boost

I suppose having it in host would be being on the save side, correct? So it might make sense to move it there just in case.

Reply:
Seems to be working, thanks @chrisburr